ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/vxlan-interface.j2

```diff
@@ -1,33 +1,57 @@
-{% if vxlan_tunnel_interface is defined %}
+{% if vxlan_tunnel_interface is arista.avd.defined %}
 
 ## VXLAN Interface
 
 ### VXLAN Interface Summary
 
-#### Source Interface: {{ vxlan_tunnel_interface.Vxlan1.source_interface }}
+#### Source Interface: {{ vxlan_tunnel_interface.Vxlan1.vxlan.source_interface }}
 
-#### UDP port: {{ vxlan_tunnel_interface.Vxlan1.vxlan_udp_port }}
+#### UDP port: {{ vxlan_tunnel_interface.Vxlan1.vxlan.udp_port | arista.avd.default('4789') }}
 
-#### VLAN to VNI Mappings
+{% if vxlan_tunnel_interface.Vxlan1.vxlan.virtual_router_encapsulation_mac_address is arista.avd.defined %}
 
-| VLAN | VNI |
-| ---- | --- |
-{%     for vlan in vxlan_tunnel_interface.Vxlan1.vxlan_vni_mappings.vlans | arista.avd.natural_sort %}
-| {{ vlan }} | {{ vxlan_tunnel_interface.Vxlan1.vxlan_vni_mappings.vlans[vlan].vni }} |
+#### EVPN MLAG Shared Router MAC : {{ vxlan_tunnel_interface.Vxlan1.vxlan.virtual_router_encapsulation_mac_address }}
+
+{% endif %}
+
+#### VLAN to VNI and Flood List Mappings
+
+| VLAN | VNI | Flood List |
+| ---- | --- | ---------- |
+{%     for vlan in vxlan_tunnel_interface.Vxlan1.vxlan.vlans | arista.avd.natural_sort %}
+| {{ vlan }} | {{ vxlan_tunnel_interface.Vxlan1.vxlan.vlans[vlan].vni }} | {{ vxlan_tunnel_interface.Vxlan1.vxlan.vlans[vlan].flood_vteps | join('<br/>') if vxlan_tunnel_interface.Vxlan1.vxlan.vlans[vlan].flood_vteps is arista.avd.defined else 'N/A' }} |
```

Comment:
default value should be `'-'` to be consistent with other templates.
We are trying to move away from inline logic to ease readability.
I suggest 
```jinja
{%     for vlan in vxlan_tunnel_interface.Vxlan1.vxlan.vlans | arista.avd.natural_sort %}
{%         set row_flood_vteps = vxlan_tunnel_interface.Vxlan1.vxlan.vlans[vlan].flood_vteps | arista.avd.default(['-']) | join('<br/>')
| {{ vlan }} | {{ vxlan_tunnel_interface.Vxlan1.vxlan.vlans[vlan].vni }} | {{ row_flood_vteps }}
```

Reply:
addressed with commit f576e6a